iothub/device/src/DeviceClient.cs

```diff
@@ -22,6 +22,11 @@ public class DeviceClient : IDisposable
         /// </summary>
         public const uint DefaultOperationTimeoutInMilliseconds = 4 * 60 * 1000;
 
+        /// <summary>
+        /// The object serializer used by the <see cref="PayloadConvention"/> set in <see cref="ClientOptions.PayloadConvention"/>. If no <see cref="PayloadConvention"/> is set this will default to <see cref="NewtonsoftJsonObjectSerializer"/>.
+        /// </summary>
+        public ObjectSerializer ObjectSerializer => InternalClient.ObjectSerializer;
```

Comment:
q - for parity, should we also provide a getter for the encoder?

Reply:
I thought about this but I don't think the customer will need to access the encoder for anything. If I were to do that maybe just being able to access the PayloadConvention.

My thought was to keep the overall PayloadConvention methods away from the customer since the convention should just be used to spit out the encoded bytes and deal with the most used part of serialization.